cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -349,22 +365,21 @@ describe('Data Table', () => {
     headCellsTooltips.forEach((isVisible, index) => {
       const assertion = !isVisible ? 'not.exist' : 'exist';
 
-      it(`Tooltip element ${index} should ${
-        !isVisible ? 'not' : ''
-      } exist as the label is ${!isVisible ? 'not' : ''} truncated`, () => {
-        cy.get(`[data-cy='data-table-truncation'] .${DATA_TABLE_CLASSES.HEAD}`)
-          .find(`.${DATA_TABLE_CLASSES.CELL}`)
-          .eq(index)
-          .children()
-          .first()
-          .as('trigger')
-          .trigger('mouseenter')
-          .then(() => {
-            cy.get('@trigger')
-              .find(`.${TOOLTIP_CLASSES.TOOLTIP}`)
-              .should(assertion);
-          });
-      });
+      it(`Tooltip element ${index} should ${!isVisible ? 'not' : ''
```

Comment:
i'd name this in a more descriptive way, e.g. existsAssertion, or isVisibleAssertion